terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1 +1,62 @@
-# placeholder file
\ No newline at end of file
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+
+# TODO Check not required (not there in OE Web Staging config)
+# 
+# aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+# remote_state_bucket = "ch-development-terraform-state-london"
+# deploy_to = "development"
+# state_prefix = "env:/development"
+# kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
+
+# service configs
+api_local_url = "https://private-api-staging.company-information.service.gov.uk"
+cdn_host = "d6nh3dxv55e16.cloudfront.net"
+chs_url = "https://staging.company-information.service.gov.uk"
+cookie_domain = "company-information.service.gov.uk"
+piwik_site_id = "15"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+session_create = "off"
+permission_name_download = "/admin/extensions-download"
+permission_name_view = "/admin/extensions-view"
+maximum_extension_requests_per_day = "50"
+feature_request_count = "false"
+max_file_size_bytes = "4194304"
+show_service_unavailable_page = "off"
+too_soon_days_before_due_date = "60"
+max_extension_period_in_months = "12"
+log_level = "debug"
+cookie_name = "__SID"
+
+# TODO CiDev setting was 1200 but set to this value (3600) in Mesos Staging config
+default_session_expiration = "3600"
+
+# TODO Needed? Not present in Mesos Staging config and variable only defined in the terraform but not used anywhere in the code that I can see
+#
+# cookie_expiration_in_seconds = "3600"
+
+
+# TODO Needed? Not present in Mesos Staging config and variable only defined in the terraform but not used anywhere in the code that I can see
+#
+# cookie_secret_flag = "0"
+
+
+# TODO Not present in Mesos Staging config (log_level isn't neither, but I think default value of 'debug' is being applied there,
+#      so making it explicit is good - that's what we did with OE Web). I just don't think human_log needs setting, if we agree? 
+#
+# human_log = "1"
+
+
+# TODO Needed? Present in Mesos Staging config, variable already defined in the terraform but not used anywhere in the code that I can see
+#
+# export AUTHENTICATION_MIDDLEWARE=on
```

Comment:
could be needed by imported libraries, if it is in mesos config i'd keep them in this config too

Reply:
Makes sense. It's not defined in the CiDev vars file but it is in the CiDev Mesos config too, so I think we missed that.

Reply:
See comment below - looks like this isn't needed, so planning to remove it.